src/components/common/ProductBox/ProductBox.module.scss

```diff
@@ -31,6 +39,12 @@
     .buttons {
       display: flex;
       justify-content: space-between;
+      position: absolute;
+      top: 89%;
```

Comment:
89% to wartość na oko, zmieni się tekst lub padding przycisku i trzeba bedzie zmieniać tą wartość, lepiej użyć tutaj `bottom`, ponieważ tak na prawdę ustawiasz te przyciski względem dolnej krawędzi.